src/transformers/convert_slow_tokenizer.py

```diff
@@ -1571,10 +1574,18 @@ def converted(self) -> Tokenizer:
             ]
         )
         tokenizer.decoder = decoders.ByteLevel()
+        tokenizer.add_special_tokens(self.special_tokens)
+
         tokenizer.post_processor = processors.ByteLevel(trim_offsets=False)
+
         return tokenizer
 
 
+TIKTOKEN_CONVERTERS = {
+    "LlamaTokenizerFast": TikTokenConverter,
+    "PreTrainedTokenizerFast": TikTokenConverter,
+}
+
```

Comment:
Not 100% sure we need a mapping for now, as we want it to be "general" but if we do take into account the special tokens, then good idea.
Either we keep it + add the special tokens of Llama automatically, (but people will ask to add new converters) or we remove it! 

Let's also update our documentation on loading tokenizers!

Reply:
okay I took out the converters!